sa.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: http://json-schema.org/draft-07/schema
+title: GA4GH-SA-Definitions
+type: object
+definitions:
+  SequenceFeature:
+    properties:
+      id:
+        type: string
+      location:
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/main/schema/vrs.yaml#definitions/Location"
+      seqOntologyRef:
```

Comment:
Is this the SO term? Or the string? Is this component capable of modelling other ontological links? Is SO special?

Reply:
I think we would like to enforce SO terminology for sequence features (junctions, regions, etc.), but there are definitely other ontologies/terminologies/nomenclatures that we'll eventually want to support (HUGO for genes, possibly ECO for evidence codes, etc.) but I don't think they're directly relevant to the "SequenceFeature" object. 

Gene names seem to generally be stored separately from the sequence feature "gene". Do you feel that we need to have the gene named at this same level or are you ok with it being identified a bit deeper in the structure @andrewyatz?